packages/dom-adapters/src/utils/index.ts

```diff
@@ -1,2 +1,14 @@
 export * from './getAbsoluteRangeOffset.js';
 export * from './getRelativeIndex.js';
+
+export enum NativeInput {
+  Textarea = 'TEXTAREA',
+  Input = 'INPUT',
```

Comment:
There are some non-text input types, but I think we shouldn't worry about them. Just throw an error if a tool tries to attach such input to adapter. Those inputs should be treated as value nodes and handled on a tool side.

What do you think @neSpecc @ilyamore88 ?

Reply:
Yes. But it should be handled explicitly. Here you can check the input types supporting `selectionStart`: https://html.spec.whatwg.org/multipage/input.html#concept-input-apply:~:text=·-,selectionStart,-·

Reply:
Still relevant

Reply:
![image](https://github.com/user-attachments/assets/736d1531-7fcf-433e-bca0-caada677bbaf)
handled